molconfgen/output.py

```diff
@@ -0,0 +1,114 @@
+# Richard Richardson wrote this ;)
+# Function to add a box to the output of
+# sampler.generate_conformers
+
+import MDAnalysis as mda
+import MDAnalysis.transformations
+import numpy as np
+from tqdm import tqdm
+
+
+def largest_r(ag):
+    """Find the largest radius to enclose 'ag'.
+
+    The function iterates over the whole trajectory associated with 'ag'
+    and returns the maximum radius.
+
+    Arguments
+    ---------
+    ag : AtomGroup
+        'ag' contains the molecule of interest
+
+    Returns
+    -------
+    r : float
+    """
+    u = ag.universe
+    r = np.max([ag.bsphere()[0] for ts in u.trajectory])
+    return r
+
+
+def write_pbc_trajectory(u, filename, l=10, box=None):
+    """Define the box for a trajectory and write to a file.
+
+    The function defines a box for the trajectory associated with 'u' and
+    writes it to a file. The default option is to write the trajectory to
+    a file without a box.
+
+    This is intended to be used with the output of molconfgen's
+    sampler.generate_conformers, but it is general enough to use with any
+    universe that contains a molecule and trajectory
+
+    Arguments
+    ---------
+    u : MDAnalysis universe
+        contains molecule of interest and a trajectory
+    l : int or float
```

Comment:
rename – `l` is ambiguous and not clear, e.g. `scale`.

Just make it a float, and if someone passes an int it will still work.